### PR TITLE
Mar googlesheetsapi

### PR DESCRIPTION
**Pull request by mrodriguez3313**

### CODE REVIEW

File: website/core/utils.py [hunk 1]
```diff
@@ -73,9 +73,9 @@ def is_url(url):
 
 def load_data(path):
```
Comment (alancastilleja) [inline website/core/utils.py:74]: pretty confused on what this does. Can you explain it over a call?

File: website/static/style.css [hunk 2]
```diff
@@ -153,8 +146,6 @@ h1 {
   top: 50%;
   -ms-transform: translateY(-50%);
   transform: translateY(-50%);
-  /* bottom: 4.5rem; */
-  /* z-index: 10; */
 }
 
 @media (max-width: 600px) {
```
Comment (alancastilleja) [inline website/static/style.css:151]: what does this do? is this for going into mobile?

File: website/templates/States_list.html [hunk 3]
```diff
@@ -5,7 +5,7 @@ <h1 class="font-weight title_blm">State List</h1>
 </div>
 
 <div class="container">
-  <h3 class="resource"><a style="color: white !important;" href="https://docs.google.com/forms/d/e/1FAIpQLSejKUgeq_edAGw43KlcmZQPX9w-jQ4WP_h33IXg-YthxidIPw/viewform?usp=sf_link">Submit a Resource or Feedback</a></h3>
+  <h3 class="resource"><a style="color: white !important;" href="https://docs.google.com/forms/d/e/1FAIpQLSfcjmCkgQtUDVoTYI2K4LDaptkgpDo_m3U1jpNXKxmGTzsJJg/viewform?usp=sf_link">Submit a Resource or Feedback</a></h3>
```
Comment (alancastilleja) [inline website/templates/States_list.html:8]: why did the resource link change?

File: website/templates/base.html [hunk 4]
```diff
@@ -19,8 +21,8 @@
   <!--Addition that needs to be looked at-->
   <header>
     <nav class="navbar navbar-expand-lg navbar-light barstyle">
-      <a class="navbar-brand" href="{{url_for('core.index')}}">SB</a>
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+      <a class="navbar-brand" href="{{url_for('core.index')}}">SP</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" data-backdrop="false" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (alancastilleja) [inline website/templates/base.html:25]: what is data-backdrop?

**Comment by mrodriguez3313** [comment]
Yeah let's do it tomorrow. I'm in SJ so I can't rn. Let's do like 3pm your
time.

File: website/core/utils.py [hunk 5]
```diff
@@ -89,11 +89,11 @@ def load_data(path):
     data = {}
     for row in rows:
         entry_type = row['Type']
-        state = row['state']
+        state = row['State']
```
Comment (alancastilleja) [inline website/core/utils.py:92]: phone died. sorry

**Comment by mrodriguez3313** [comment]
Lol oops

File: website/core/utils.py (hunk 5 above)
Comment (mrodriguez3313) [reply website/core/utils.py:92]: oh lol I just realized I was talking to nobody haha

**Comment by alancastilleja** [comment]
ANyway, it looks good so just push it and we can move towards deployment. Also, ill make an issue for the donation thing if we really need it.

**Pull request merged**